src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
```

Comment:
It's better not to use static methods or fields unless they are constants or utility methods. In this case, `COLORS` is a constant array, so it is correctly defined as `static final`.